hail/python/hailtop/pipeline/task.py

```diff
@@ -404,6 +404,31 @@ def image(self, image):
         self._image = image
         return self
 
+    def always_run(self, always_run = True):
```

Comment:
Change looks good, will approve, but no spaces around equals in default arguments.  You're getting a lint error.